strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/maven/MavenArtifactController.java

```diff
@@ -213,7 +218,48 @@ public void download(@ApiParam(value = "The storageId", required = true)
 
         logger.debug("Download succeeded.");
     }
-
+    
+    @ApiOperation(value = "Used to retrieve headers of an artifact", position = 2)
+    @ApiResponses(value = { @ApiResponse(code = 200, message = ""),
+                            @ApiResponse(code = 400, message = "An error occurred.") })
+    @PreAuthorize("hasAuthority('ARTIFACTS_RESOLVE')")
+    @RequestMapping(value = { "{storageId}/{repositoryId}/{path:.+}" }, method = RequestMethod.HEAD)
+    public void getHeaders(@ApiParam(value = "The storageId", required = true)
+                           @PathVariable String storageId,
+                           @ApiParam(value = "The repositoryId", required = true)
+                           @PathVariable String repositoryId,
+                           @RequestHeader HttpHeaders httpHeaders,
+                           @PathVariable String path,
+                           HttpServletRequest request,
+                           HttpServletResponse response)
+            throws Exception
+    {
+        logger.debug("Requested headers for /" + storageId + "/" + repositoryId + "/" + path + ".");
+        
+        RepositoryPath resolvedPath = getArtifactManagementService().getPath(storageId, repositoryId, path);
+        
+        logger.debug("Resolved path : " + resolvedPath);
+        if(resolvedPath == null)
+        {
+            response.setStatus(HttpStatus.NOT_FOUND.value());
+            return;
+        }
+        
+        ArtifactInputStream ais = (ArtifactInputStream) Files.newInputStream(resolvedPath);
+        Repository repository = configurationManager.getRepository(storageId, repositoryId);
+        RepositoryInputStream is =  RepositoryInputStream.of(repository, path, ais);
+      
+        RepositoryFileAttributes fileAttributes = Files.readAttributes(resolvedPath, RepositoryFileAttributes.class);
+        
+        ArtifactControllerHelper.setHeadersForChecksums(is, response);
+        response.setHeader("Accept-Ranges", "bytes");
+        response.setHeader("Content-Length", String.valueOf(fileAttributes.size()));
+        response.setHeader("Last-Updated", fileAttributes.lastModifiedTime().toString());
+        setMediaTypeHeader(path, response);
+        
+        logger.debug("Header Download succeeded.");
```

Comment:
`logger.debug("Header Download succeeded.");`

Please drop this debug line. Thanks.